### ISSUE TITLE
Contribution with ShareLaTeX

### ISSUE THREAD

**Issue by blegat** [Gp2mv3/Syntheses#358]
ShareLaTeX now has Github support. It would be nice to add to the documentation how to do Pull Requests from ShareLaTeX, it would remove the technical barrier to new contributions.

**Comment by fthuin** [comment]
Will it be free? Because €14/month to avoid using git commands seems too expensive

![selection_091](https://cloud.githubusercontent.com/assets/7595324/22022754/9bf65c88-dcc3-11e6-9a27-ca96af2efe14.png)

![selection_092](https://cloud.githubusercontent.com/assets/7595324/22022930/4fc4e9f0-dcc4-11e6-8ac9-b165cb88d1de.png)

**Comment by blegat** [comment]
Ah zut :( Maybe the user can clone the project on their ShareLaTeX and do pull request from it without the need of our project to be on ShareLaTeX ? I have no idea what is the policy but Martin Braquet is trying to make a PR from ShareLaTeX, we'll see if he succeeds :)

**Comment by MartinBraquet** [comment]
Effectivement, le partage direct via ShareLatex semble payant (mais une synchronisation avec Dropbox est possible). Je ne comptais pas synchroniser Sharelatex et Github mais simplement écrire les fichiers en .tex via Sharelatex (qui permet une visualisation très rapide du document). Je les télécharge ensuite en .tex et les ajoute sur Github. ;)

**Comment by fthuin** [comment]
Et tu arrives à charger tous les fichiers dessus ? Parce que ce repository est particulièrement gros (+2000 fichiers, +200Mo), l'offre la plus chère d'Overleaf permet max 1000 fichiers et l'offre la plus chère de Papeeria permet max 128Mo.

**Comment by MartinBraquet** [comment]
Pas sur Sharelatex puisque je ne suis pas premium...

**Comment by ldevillez** [comment]
Même si on a parrainer suffisamment de personnes on a pas accès à l'option de github. Et j'ai testé d'upload tous les fichiers des synthèses epl et il refuse à partir d'un moment (mais peut être qu'en premium la limite est plus loin)

**Comment by pgonzalezalv** [comment]
Une autre piste sinon (cf. la FAQ en bas de page) : https://www.overleaf.com/blog/195-new-collaborate-online-and-offline-with-overleaf-and-git-beta
